modules/core/src/main/java/org/apache/ignite/lang/ErrorGroup.java

```diff
@@ -106,10 +102,10 @@ public int registerErrorCode(int errorCode) {
     /**
      * Checks that the given {@code code} is registered for this error group.
      *
-     * @param code Full error code to be tested.
```

Comment:
So, initial javadoc was incorrect, and it wasn't "Full error code", but "group specific code", right?

Reply:
Yes, the initial javadoc was incorrect. Moreover, there is no test for this method... (my fault).
In any way, I removed this method because there are no tests/prod code that use it.